program-keccak-inclusion/Cargo.toml

```diff
@@ -9,11 +9,19 @@ publish = false
 
 [dependencies]
 # TODO: use workspace version, cannot override default feats though... 
-eq-common = {path = "../common", default-features = false}
+eq-common = { path = "../common", default-features = false }
 
-sp1-zkvm = {workspace = true}
-celestia-types = {workspace = true}
-nmt-rs = {workspace = true}
-tendermint = {workspace = true}
-tendermint-proto = {workspace = true}
-sha3 = {workspace = true}
\ No newline at end of file
+sp1-zkvm = { workspace = true }
+celestia-types = { workspace = true }
+nmt-rs = { workspace = true }
+tendermint = { workspace = true }
+tendermint-proto = { workspace = true }
+sha3 = { workspace = true }
+
+# Greatly reduces cycle count when executing the zkVM program
+# Also reflected in sp1/build.rs
+[profile.release]
+opt-level = 3
+lto = "fat"
+codegen-units = 1
+debug = 1
```

Comment:
this throws a wanring that it's ignored... :thinking: would be worth leaving in IMHO